tests/check/test_sinks.py

```
@@ -0,0 +1,131 @@
+            url="http://example.com/2",
+            type_test="example",
+        ),
+    ]
+    tmp_path = Path(__file__).parent / "out"
+    output_file = tmp_path / "test_results.html"
+
+    write_html(results, str(output_file))
+
+    assert output_file.exists()
+
+    with open(output_file, "r") as f:
```

Ruff

Unnecessary open mode parameters

Remove open mode parameters

(UP015)